main.cpp

```diff
@@ -98,6 +98,31 @@ struct Computer
         bool setPrice(float newPrice = 100.0f);
         //output cuda version and number of cores
         std::string outputCUDAVersionAndCores();//returns a string representing the cuda capbility 
+
+        void boostTheGraphics()
+        {
+            int testValue = 0;
+            while(isGSyncCapable)
+            {
+                testValue = 1;
+                std::cout << "Boosted!" << std::endl;
+                break;
+            }
+            std::cout << "Out of Boost" << std::endl;
+        } // part 5 begin.
+        int parallelSpeedIncreaseFactor(int desiredFactor)
+        {
+            int totalCoresProcessing = 0;
+            for(int i = 0; i <= int(price); ++i)
```

Comment:
casting has not been introduced.  And this is a C-style cast.
Remove this cast.
If a warning is caused by removing the cast, change the type of `price` or change the loop.
You will learn about proper casting techniques in Project 4 Part 3 and why C-style casts are a "code smell"
- [ ] revise